### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by satyaog**
# Description

Fix CI and warn on errors in full_simu_test until permanent fix.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] python run_tests.py

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by satyaog** [comment]
@pg2455 , It will be needed to remove python 3.8 from the required tests in develop (could also be done for master at the same time)

**Comment by pg2455** [comment]
Why is Python3.8 not supported? Is there anything specific to it that doesn't work with the current codebase?

File: setup.py [hunk 1]
```diff
@@ -25,24 +28,7 @@
     ],
     zip_safe             = False,
     python_requires      = '>=3.7.4',
-    install_requires     = [
-        "speedrun @ git+https://github.com/inferno-pytorch/speedrun.git@p2p_risk_prediction#egg=speedrun",
-        "ctt @ git+https://github.com/nasimrahaman/ctt@master#egg=ctt",
```
Comment (plstcharles) [inline setup.py:30]: the "#egg=<pkgname>" suffix might be missing from the reqs.txt file; I'm not sure what the difference is, but I've only seen examples online with the egg thing (and the <pkgname @> prefix) if it's used here

**Comment by satyaog** [comment]
@pg2455 

> Why is Python3.8 not supported? Is there anything specific to it that doesn't work with the current codebase?

[ctt](https://github.com/nasimrahaman/ctt) depends on tensorflow 2.1 which is not compatible with python 3.8.

File: setup.py (hunk 1 above)
Comment (satyaog) [reply setup.py:30]: I've used the content of setup.py in requirements.txt but I've just added a comment in the .txt to explain the format it should follow with git end points to make sure future additions don't break the install

**Comment by satyaog** [comment]
@pg2455, ping
ctt depends on tensorflow 2.1 which is not compatible with python 3.8.

**Comment by mweiss17** [comment]
It appears the tests are passing and the only changes have to do with handling results from the server which @nasimrahaman is going to fix on his end. Is there anything preventing us from merging this? I would like to get passing tests soon :)

**Comment by satyaog** [comment]
Rebased on develop and all tests passes!

**Comment by pg2455** [comment]
With #121 I think we can retain Python=3.8 in the tests. what do you think?

**Comment by obilaniu** [comment]
I would prefer it. My current Anaconda environment is Python 3.8-centric.

**Comment by satyaog** [comment]
I need ctt to launch the inference server in the tests. #121 would need to also be in ctt separating the dependencies of pytorch and tensorflow which would be better since it takes 1-2 min just to install those in the CI

**Comment by obilaniu** [comment]
@satyaog @pg2455 I would merge this one first, and now. Merging #121 first will break CI, that needs to be changed to have CI do `pip install .[ctt]`.

**Pull request merged**